pole.h

```diff
@@ -0,0 +1,8 @@
+class Pole
```

Comment:
Sorry, I wasn't there for the meeting last time; what's the purpose of this class?